mmdet/core/anchor/anchor_generator.py

```diff
@@ -1,54 +1,112 @@
+import numpy as np
 import torch
 
+from .registry import ANCHOR_GENERATORS
 
+
+@ANCHOR_GENERATORS.register_module
 class AnchorGenerator(object):
-    """
+    """Standard anchor generator for 2D anchor-based detectors
+
+    Args:
```

Comment:
Although many arguments are lists, the meanings are different. It is better to illustrate what the list stands for.